resources/public/index.html

```diff
@@ -16,10 +16,7 @@
         <script src="cljs-out/dev/main_bundle.js" type="text/javascript"></script>
         <script type="text/javascript">
          //<![CDATA[
-         com.yetanalytics.lrs_admin_ui.core.init(
-             "serverHost", "http://localhost:8080",
-             "xapiPrefix", "/xapi"
-         );
+         com.yetanalytics.lrs_admin_ui.core.init("http://localhost:8080");
          //]]>
```

Comment:
I think we can adjust the SQL LRS documentation for `LRSQL_ENABLE_STMT_HTML` then (since right now it implies that if this is false the _entire admin UI_ doesn't work).